tests/sentry/utils/sdk_crashes/test_sdk_crash_detection_dotnet.py

```diff
@@ -224,3 +224,79 @@ def test_sdk_crash_is_reported_with_unity_paths(
 
     else:
         assert mock_sdk_crash_reporter.report.call_count == 0
+
+
+@pytest.mark.parametrize(
+    ["sdk_frame_module", "system_frame_module", "detected"],
+    [
+        # Sentry.Samples.* should be filtered out (not detected as SDK crash)
+        (
+            "Sentry.Samples.AspNetCore.Mvc",
+            "System.Threading.Tasks.Task",
+            False,
+        ),
+        (
+            "Sentry.Samples.Console.Basic",
+            "Microsoft.Extensions.Logging.ILogger",
+            False,
+        ),
+        (
+            "Sentry.Samples.Maui",
+            "System.Net.Http.HttpClient",
+            False,
+        ),
+        # Regular Sentry SDK frames should still be detected
+        (
+            "Sentry.AspNetCore.SentryMiddleware",
+            "System.Threading.Tasks.Task",
+            True,
+        ),
+        (
+            "Sentry.SentryClient",
+            "Microsoft.Extensions.Logging.ILogger",
+            True,
+        ),
+    ],
+)
+@decorators
+def test_sentry_samples_filter(
```

Comment:
I think it would make sense to add a test to validate that an SDK crash within a sample app is still detected. I think it should, but I'm not 100% sure.